From ada302b1054c0620e2b39c6af6f6ffad4fcb1641 Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Fri, 27 Sep 2019 12:44:24 -0700
Subject: [PATCH 0673/1852] dt-bindings: display: xlnx: Add support for vpss ip
 version 2.2

commit 5bcfe46c0b337639391194a4e1d9dfe2bf9d2568 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds compatible string to support 2.2 version.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
index 6e83436ffaaa..3927b88be0cf 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
@@ -6,7 +6,7 @@ resolution values between 0 to 4096.
 
 Required properties:
 
-- compatible: Must be "xlnx,vpss-scaler".
+- compatible: Must be "xlnx,vpss-scaler-2.2" or "xlnx,vpss-scaler".
 
 - reg: Physical base address and length of registers set for the device.
 
-- 
2.31.1

